spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/converters/PubSubMessageConverter.java

```diff
@@ -26,40 +30,58 @@
 import org.springframework.messaging.support.GenericMessage;
 
 /**
- * Generates {@link PubsubMessage} from {@link Message} and vice-versa.
+ * Generates {@link PubsubMessage} from {@link Message} with payload type of
+ * {@link String} and vice-versa. When converting to String, it defaults to using UTF-8
+ * @{link Charset}.
  *
- * @author João André Martins
+ * @author Ray Tsang
```

Comment:
That's rude. You are not only an author of the class.
So, your name should be just added alongside with others, not replaces.
Thanks

Reply:
eak, actually, this was a brand new converter and the original one was removed, so Git thought it was a rename :/ didn't mean to be rude at all!